code/modules/antagonists/slaughter/slaughter.dm

```diff
@@ -90,25 +94,23 @@
 	user.visible_message("<span class='warning'>[user] raises [src] to their mouth and tears into it with their teeth!</span>", \
 						 "<span class='danger'>An unnatural hunger consumes you. You raise [src] your mouth and devour it!</span>")
 	playsound(user, 'sound/magic/demon_consume.ogg', 50, 1)
-	for(var/obj/effect/proc_holder/spell/knownspell in user.mind.spell_list)
-		if(knownspell.type == /obj/effect/proc_holder/spell/bloodcrawl)
-			to_chat(user, "<span class='warning'>...and you don't feel any different.</span>")
-			qdel(src)
-			return
+	if(user.bloodcrawl == BLOODCRAWL)
+		to_chat(user, "<span class='warning'>...and you don't feel any different.</span>")
```

Comment:
Mark your changes ![like this](https://github.com/yogstation13/tg-rebase/blob/master/yogstation/README.md)